### PR TITLE
[Update] action-blocks.md: Added notice on actions not supported in Action Block

### PR DESCRIPTION
**Pull request by Rodriq**
### Description
notice that some actions are not available in the action block

## Type of change
- [ ] Typo fix 
- [ ] New feature 
- [X] Enhancement to current docs
- [ ] Removed outdated references
- [ ] Update assets

### CODE REVIEW

**Review by PoojaB26 — CHANGES_REQUESTED** [review]
Hi @Rodriq, thanks for providing a PR. However, we use a different syntax for callouts in this project. Its called Admonitions, and please see the s[yntax for adding a notes admonition](https://flutterflow.notion.site/Docs-Contribution-Guidelines-42829c305df441f6b061b4095abac8a8#84fead23023e4b7b806272b7e3b349fd). Here is the [contribution guidelines link](https://flutterflow.notion.site/Docs-Contribution-Guidelines-42829c305df441f6b061b4095abac8a8) (also linked in README now)

File: docs/resources/control-flow/functions/action-blocks.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Action Blocks have different scopes, which determine their availability:
 | **Page Level Action Blocks**     | Restricted to the page in which they were created. These can access the state variables available in their scope, such as page state variables, as well as variables above their scope, such as [App State variables](../../../resources/data-representation/app-state.md). | Page Level Action Blocks can access page state variables and App State variables.                                                                              |
 | **Component Level Action Blocks**| Restricted to the component in which they were created. These can access the state variables available in their scope, such as component state variables, as well as variables from higher scopes, like page and App State variables. | Component Level Action Blocks can access component state variables, page state variables, and App State variables.                                             |
 
+> :information_source: **Note**: Some actions are not supported and cannot be used in an Action Block, these actions are hidden there by default.
```
Comment (PoojaB26) [inline docs/resources/control-flow/functions/action-blocks.md:19]: Can we add 1-2 example Actions here that are not available in Action Blocks to provide more clarity?

Comment (Rodriq) [reply docs/resources/control-flow/functions/action-blocks.md]: Noted.

File: docs/resources/control-flow/functions/action-blocks.md [hunk 2]
```diff
@@ -16,6 +16,10 @@ Action Blocks have different scopes, which determine their availability:
 | **Page Level Action Blocks**     | Restricted to the page in which they were created. These can access the state variables available in their scope, such as page state variables, as well as variables above their scope, such as [App State variables](../../../resources/data-representation/app-state.md). | Page Level Action Blocks can access page state variables and App State variables.                                                                              |
 | **Component Level Action Blocks**| Restricted to the component in which they were created. These can access the state variables available in their scope, such as component state variables, as well as variables from higher scopes, like page and App State variables. | Component Level Action Blocks can access component state variables, page state variables, and App State variables.                                             |
 
+:::info[Note]
+Some actions are not supported and cannot be used in an Action Block, these actions are hidden in the action block editor by default.
```
Comment (PoojaB26) [inline docs/resources/control-flow/functions/action-blocks.md:20]:
```suggestion
Certain actions are not supported and cannot be used in an Action Block. By default, these actions are hidden in the Action Block Editor. 
```

File: docs/resources/control-flow/functions/action-blocks.md [hunk 3]
```diff
@@ -16,6 +16,10 @@ Action Blocks have different scopes, which determine their availability:
...
+For example actions like all the actions under the **Firebase Authentication** category, **Start Periodic Action**, **Upload Data**, etc
```
Comment (PoojaB26) [inline docs/resources/control-flow/functions/action-blocks.md:21]:
```suggestion
For example, actions under the **Firebase Authentication** category, **Start Periodic Action**, **Upload Data**, and others.
```

File: docs/resources/control-flow/functions/action-blocks.md [hunk 4]
```diff
@@ -16,6 +16,10 @@ Action Blocks have different scopes, which determine their availability:
 | **Page Level Action Blocks**     | Restricted to the page in which they were created. These can access the state variables available in their scope, such as page state variables, as well as variables above their scope, such as [App State variables](../../../resources/data-representation/app-state.md). | Page Level Action Blocks can access page state variables and App State variables.                                                                              |
 | **Component Level Action Blocks**| Restricted to the component in which they were created. These can access the state variables available in their scope, such as component state variables, as well as variables from higher scopes, like page and App State variables. | Component Level Action Blocks can access component state variables, page state variables, and App State variables.                                             |
 
+:::info[Note]
```
Comment (PoojaB26) [inline docs/resources/control-flow/functions/action-blocks.md:19]:
```suggestion
:::note[Unsupported Actions in Action Blocks]
```

**Review by PoojaB26 — CHANGES_REQUESTED** [review]
Suggested some language and title changes, please check and Accept Changes if that works with you!

Please accept the suggestions in a single commit instead of 3 different commits. 
Check [pt 3.2 in the Github PR guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes).

**Comment by Rodriq** [comment]
@PoojaB26 I could open a new PR if needed.
I accepted your suggestions one after the other.

**Review by PoojaB26 — APPROVED** [review]
No worries, you can take care of it in a future PR so all such suggested changes can be part of one single commit. I'm approving it now and merging, thanks!!!

**Pull request merged**